### ISSUE TITLE
First party support for Compose UI

### ISSUE THREAD

**Issue by saket** [google/flexbox-layout#632]
- [X] I have searched [existing issues](https://github.com/google/flexbox-layout/issues) and confirmed this is not a duplicate

While Compose UI has good interop APIs to work with Views, it'd be superb if `flexbox-layout` had a first party support for compose. 

```kotlin
FlexboxRow(
  modifier = Modifier.fillMaxWidth(),
  flexWrap = FlexWrap.Wrap,
  divider = { … },
) {
  Text("…")
  Text("…")
  Text("…")
}
```

## Version of the flexbox library
`3.0.0`

**Comment by mtotschnig** [comment]
Compose has FlowRow and FlowColumn, which should cover most of the functionality this library provides. Am I missing something?

**Comment by saket** [comment]
You're correct that it covers many of the functionalities, but not all of them. IIRC there's no equivalent for `FlexDirection` and `FlexWrap`.

**Comment by consultors** [comment]
https://github.com/google/flexbox-layout.git
